In [1]:
import os
import numpy as np
import pandas as pd

In [2]:
TEST_PATH = '../input/test'

In [4]:
%%time
file_sizes = []
for f in os.listdir(TEST_PATH):
    if f.endswith(".csv"):
        df = pd.read_csv(os.path.join(TEST_PATH, f))
        file_sizes.append(df.shape)

CPU times: user 25.6 s, sys: 5.26 s, total: 30.9 s
Wall time: 1min 1s


In [5]:
print("Total {} testing files.".format(len(file_sizes)))

Total 2624 testing files.


In [9]:
for shape in file_sizes:
    try:
        assert shape == (150000, 1), "What the fuck is this?"
    except:
        print(shape)

##### The above blocks show that there are `2624` testing examples and each of them contains `150000` sample points.